lib/pbench/server/api/resources/users_api.py

```diff
@@ -11,6 +11,57 @@
 from pbench.server.api.auth import Auth
 
 
+class PasswordRecovery(Resource):
+    def __init__(self, logger, auth, mail):
+        self.logger = logger
+        self.auth = auth
+        self.mail = mail
```

Comment:
This could just as easily be `self.mail = Mail(app)` if you replace your `mail` instance with `app` in the constructor parameter list.